components/resources/commands.grdp

```diff
@@ -0,0 +1,487 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grit-part>
+  <!--
+    Please reuse existing strings as much as possible rather than adding new
+    strings here.
+  -->
+  <message name="IDS_IDC_BACK">
```

Comment:
I think it's fine as-is, but we usually only use title case on macOS

We could have a `<if expr="use_titlecase">` around these... but then it sucks duplicating the strings (minus casing) in a <if expr="not use_titlecase">` block

Reply:
Honestly, I think the visibility on these is low enough that it's probably okay to just use one case. Happy to change everything to lower or leave as title, up to you.

Reply:
I've converted to sentence case :smile:

Reply:
Titlecase / non-titlecase only applies to app and context menus, right? Whereas this is for the shortcut customization in settings, and we don't usually adjust casing on settings per-platform.